package.json

```diff
@@ -43,5 +47,14 @@
     "run-waterfall": "^1.1.6",
     "serialize-error": "^2.1.0",
     "uid-safe": "^2.1.5"
+  },
+  "prettier": {
```

Comment:
Is prettier even used in this project?

Reply:
it is not…

Reply:
Added to help other developers (including myself) write their JS in a way that's consistent with the existing project, rather than having a mix of styles.